packages/-adapter-encapsulation-test-app/.travis.yml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
we don't need this config.

this file also exists in the serializer encapsulation test app but should be deleted there as well.